es-es/racket-es.html.markdown

```diff
@@ -0,0 +1,688 @@
+---
+language: racket
+filename: learnracket-es.rkt
+contributors:
+  - ["th3rac25", "https://github.com/voila"]
+  - ["Eli Barzilay", "https://github.com/elibarzilay"]
+  - ["Gustavo Schmidt", "https://github.com/gustavoschmidt"]
+  - ["Duong H. Nguyen", "https://github.com/cmpitg"]
+  - ["Keyan Zhang", "https://github.com/keyanzhang"]
+translators:
+    - ["Carlos Roman", "https://github.com/carlochess"]
+lang: es-es
+---
+Racket es un lenguaje de propósito general, multiparadigma que hace parte de la familia Lisp/Scheme
+Racket is a general purpose, multi-paradigm programming language in the Lisp/Scheme family.
+
+
+La retroalimentación de bienvenida, puedes encontrarme en [@th3rac25](http://twitter.com/th3rac25) or th3rac25 [at] [google's email service]
```

Comment:
> La retroalimentación de bienvenida

This does not make sense to me. A better translation would be “Agradecemos tus opiniones”.